src/main/resources/application.yaml

```diff
@@ -28,3 +28,17 @@ spring:
       poolName: FeatureToggleHikariCP
       maxLifetime: 7200000
       connectionTimeout: 30000
+
+webconsole:
+  users:
+    admins:
+      - username: ${TEST_ADMIN_USERNAME}
+        password: ${TEST_ADMIN_PASSWORD}
+    editors:
+      - username: ${TEST_EDITOR_USERNAME}
+        password: ${TEST_EDITOR_PASSWORD}
+    readers:
+      - username: ${TEST_READ_USERNAME}
+        password: ${TEST_READ_PASSWORD}
```

Comment:
shouldn't this be in tests directory?

Reply:
nope functional tests are executed on actual instance.